docs/source/performance.rst

```diff
@@ -98,14 +100,14 @@ However, know that 16-bit and multi-processing (any DDP) can have issues. Here a
 
 1. `CUDA error: an illegal memory access was encountered <https://github.com/pytorch/pytorch/issues/21819>`_.
```

Comment:
I can't mark it, but one sentence above it says: "Use 16-bit to decrease the memory"
this should be "memory consumption".